beacon-chain/db/operation.go

```diff
@@ -0,0 +1,39 @@
+package db
+
+import (
+	"github.com/boltdb/bolt"
+	"github.com/gogo/protobuf/proto"
+	pb "github.com/prysmaticlabs/prysm/proto/beacon/p2p/v1"
+	"github.com/prysmaticlabs/prysm/shared/hashutil"
+)
+
+// SaveExit puts the exit request into the beacon chain db.
+func (db *BeaconDB) SaveExit(exit *pb.Exit) error {
+	hash, err := hashutil.HashProto(exit)
+	if err != nil {
+		return err
+	}
+	encodedState, err := proto.Marshal(exit)
+	if err != nil {
+		return err
+	}
+
+	return db.update(func(tx *bolt.Tx) error {
+		a := tx.Bucket(operationsBucket)
+
+		return a.Put(hash[:], encodedState)
+	})
+}
+
+// HasExit checks if the exit request exists.
+func (db *BeaconDB) HasExit(hash [32]byte) bool {
+	exists := false
+	// #nosec G104
```

Comment:
Why can you ignore this error? Please add comment any time you ignore a security warning.